code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -180,12 +180,12 @@
 		return
 	M.current.visible_message("<span class='warning'>[M.current.name]'s eyes flash briefly as he stares into [C.name]'s eyes</span>")
 	M.current.verbs -= /client/proc/vampire_hypnotise
-	spawn(1800)
+	spawn(2400)
```

Comment:
While you're at it : change `2400` for `240 SECONDS` or better yet, `4 MINUTES`
Never too early to learn good formatting habits